### PR TITLE
Handle whitespace

### LINKED ISSUE #19: Zip creation errors when path has spaces in it
Zip creation failed: Failed to run '/Library/Java/JavaVirtualMachines/default/Contents/Home/bin/jar cvf /Users/bfrenkel/Documents/InstantGames/Google Play Instant Sample/SphereBlastPCV1.zip -C /var/folders/js/zwj26yx53j1bw9ywks160ryr00772k/T/ base.apk'
stdout:
added manifest
adding: base.apk(in = 16531357) (out= 15605940)(deflated 5%)

stderr:
Play : no such file or directory
Instant : no such file or directory
Sample/SphereBlastPCV1.zip : no such file or directory

exit code: 1

### PR DESCRIPTION
**Pull request by thanwah**
Update all command line calls to wrap file/directory paths in double quotes
if the path contains at least one space.

Update the Build Settings screen to trim whitespace in text fields when
clicking "Save".

Fixes #19

### CODE REVIEW

File: GooglePlayInstant/Editor/ApkSigner.cs [hunk 1]
```diff
@@ -107,7 +111,10 @@ public static bool Sign(string apkPath)
 
             var arguments = string.Format(
                 "-jar {0} sign --ks {1} --ks-key-alias {2} --pass-encoding utf-8 {3}",
-                GetApkSignerJarPath(), keystoreName, keyaliasName, apkPath);
+                CommandLine.QuotePathIfNecessary(GetApkSignerJarPath()),
```
Comment (leegao) [inline GooglePlayInstant/Editor/ApkSigner.cs]: minor nit, and I don't feel too strongly about this: apksigner's CLI syntax is a little obtuse sometimes, the verify above is fine, but this one is a little hard to grok, let's add a comment here to explain what the different flags do

File: GooglePlayInstant/Editor/PlayInstantBuildConfiguration.cs [hunk 2]
```diff
@@ -50,39 +50,40 @@ private class Configuration
         /// <summary>
         /// Optional field used to prevent removal of required components when building with engine stripping.
         /// <see cref="https://docs.unity3d.com/ScriptReference/BuildPlayerOptions-assetBundleManifestPath.html"/>
+        /// Never null.
```
Comment (leegao) [inline GooglePlayInstant/Editor/PlayInstantBuildConfiguration.cs]: is there a [NonNull] or [NotNull] attribute that's applicable here?

Comment (thanwah) [reply GooglePlayInstant/Editor/PlayInstantBuildConfiguration.cs]: Good question. There was a [NotNull] but it's not recommended anymore.
https://stackoverflow.com/questions/30896853/notnull-attribute

File: GooglePlayInstant/Editor/ApkSigner.cs (hunk 1 above)
Comment (thanwah) [reply GooglePlayInstant/Editor/ApkSigner.cs]: Good suggestion. Done.

**Pull request merged**